### PR TITLE
feat: Coding Standards deletion IO-324

### PR DESCRIPTION
**Pull request by nicklem**
In the context of allowing users to manage multiple coding standards, update the [relevant docs page](https://docs.codacy.com/organizations/using-a-coding-standard/) with a section on coding standard deletion.

### Note

This should go live after https://github.com/codacy/docs/pull/1577

### CODE REVIEW

File: docs/organizations/using-a-coding-standard.md [hunk 1]
```diff
@@ -94,6 +94,16 @@ To edit an existing coding standard or change the repositories that follow that
 !!! tip
     To ensure that all new repositories automatically follow the coding standard, [set the coding standard as default](#set-default).
 
+## Deleting a coding standard {: id="deleting"}
+
+To delete a coding standard:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Click the trash can icon on the coding standard card and confirm by clicking **Delete coding standard**.
```
Comment (nicklem) [inline docs/organizations/using-a-coding-standard.md:103]: Are we OK merging these closely related steps in one instruction?

Comment (prcr) [reply docs/organizations/using-a-coding-standard.md:103]: Yes, and I wouldn't even mention the confirmation step as it should be obvious/straightforward for users (in other situations where there's a confirmation prompt our docs don't mention it as well).

Comment (nicklem) [reply docs/organizations/using-a-coding-standard.md:103]: Thanks, @prcr! I'd still like to leave the "and confirm" fragment at the end of the sentence to differentiate it from a previous step that does not require confirmation (changing the default CS) and to reassure the user that this destructive action does require 2 clicks.

Comment (nicklem) [reply docs/organizations/using-a-coding-standard.md:103]:
```suggestion
1.  Click the trash can icon on the coding standard card and confirm.
```

**Comment by nicklem** [comment]
Closing this one after merging these changes with #1577 since the features will go live together.

**Pull request closed without merging**